tests/unit_tests/training/utils/test_train_utils.py

```
@@ -1540,6 +1541,64 @@ def test_report_throughput_negative_elapsed_wct(self, mock_print_rank_0):
         assert "skipping throughput calculation" in warning_message
         assert f"iteration {iteration}" in warning_message
 
+    @mock.patch("megatron.bridge.training.utils.train_utils.print_rank_0")
+    def test_report_throughput_resume_from_ckpt(self, mock_print_rank_0):
```

Ruff (0.15.1)

[warning] Unused method argument: `mock_print_rank_0`

(ARG002)